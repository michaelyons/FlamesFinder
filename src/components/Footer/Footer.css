footer {
  display: flex;
  justify-content: space-around;
  align-items: center;
  position: fixed;
  height: 3rem;
  bottom: 0;
  width: 100vw;
  background: slategrey;
}

.home-link {
  display: flex;
  justify-content: center;
  align-items: center;
  font-size: 2rem;
  color: aqua;
  cursor: pointer;
}
