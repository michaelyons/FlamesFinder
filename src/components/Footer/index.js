import React from 'react';
import { NavLink } from 'react-router-dom';

import './Footer.css';

const Footer = () => {
  return (
    <footer>
      <NavLink exact to={'/'} className="home-link">
        <p>
          <i className="fas fa-home" />
        </p>
      </NavLink>
    </footer>
  );
};

export default Footer;
