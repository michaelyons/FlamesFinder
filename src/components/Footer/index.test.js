import React from 'react';
import { shallow } from 'enzyme';

import Footer from './index';

describe('Footer', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<Footer />);
  });

  it('should match the snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
