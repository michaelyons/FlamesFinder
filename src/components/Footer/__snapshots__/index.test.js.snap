// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer should match the snapshot 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Footer />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <NavLink
        activeClassName="active"
        aria-current="page"
        className="home-link"
        exact={true}
        to="/"
>
        <p>
                <i
                        className="fas fa-home"
                />
        </p>
</NavLink>,
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "function",
      "props": Object {
        "activeClassName": "active",
        "aria-current": "page",
        "children": <p>
          <i
                    className="fas fa-home"
          />
</p>,
        "className": "home-link",
        "exact": true,
        "to": "/",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <i
            className="fas fa-home"
/>,
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "className": "fas fa-home",
          },
          "ref": null,
          "rendered": null,
          "type": "i",
        },
        "type": "p",
      },
      "type": [Function],
    },
    "type": "footer",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <NavLink
          activeClassName="active"
          aria-current="page"
          className="home-link"
          exact={true}
          to="/"
>
          <p>
                    <i
                              className="fas fa-home"
                    />
          </p>
</NavLink>,
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "activeClassName": "active",
          "aria-current": "page",
          "children": <p>
            <i
                        className="fas fa-home"
            />
</p>,
          "className": "home-link",
          "exact": true,
          "to": "/",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <i
              className="fas fa-home"
/>,
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "className": "fas fa-home",
            },
            "ref": null,
            "rendered": null,
            "type": "i",
          },
          "type": "p",
        },
        "type": [Function],
      },
      "type": "footer",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
    "attachTo": undefined,
    "hydrateIn": undefined,
  },
}
`;
